README.md

```diff
@@ -1,5 +1,30 @@
 # Onigumo #
 
+## About
+
+Onigumo slouží k procházení webových aplikací či statických webových stránek.
+Obsah a případně další příslušná metatdata konkrétní *url* adresy uloží do
+strukturované podoby dat (např. JSON nebo YAML), které je vhodné
+pro další strojové zpracování. K dosažení tohoto cíle je `Onigumo` rozdělen
```

Comment:
I‘d refrain from enclosing the project name in backticks. It is not a piece of code.

```suggestion
pro další strojové zpracování. K dosažení tohoto cíle je Onigumo rozdělen
```